### DISCUSSION TITLE
Support for Vue Devtools

### DISCUSSION THREAD

**Discussion by skamansam** [vuepress/core#62] [Q&A] [answered]
Vue Devtools has become a central part of my developer experience, especially while debugging SFCs. Is there any planned support for using Vue Devtools? If not, could you explain what the limitation is?

**Comment by meteorlxy** [comment, marked as answer]
Please check if you are using the correct version of vue-devtools.

Only version 6+ supports vue3 projects (currently v6.0.0-beta.7)
